angular/projects/spark-angular/src/lib/components/sprk-footer/sprk-footer.component.spec.ts

```diff
@@ -40,7 +40,7 @@ describe('SprkFooterComponent', () => {
     component.additionalClasses = 'sprk-u-pam sprk-u-man';
     fixture.detectChanges();
     expect(component.getClasses()).toEqual(
-      'sprk-o-CenteredColumn sprk-o-Stack sprk-o-Stack--misc-b sprk-u-pam sprk-u-man'
+      'sprk-o-CenteredColumn sprk-o-Stack sprk-o-Stack--misc-b sprk-c-Footer sprk-c-Footer--has-dark-bg sprk-u-pam sprk-u-man'
```

Comment:
hey @guern1kn , is the new footer only going to have a dark background now? If so, why didn't we update the sprk-c-Footer to be the dark color instead of applying a modifier?

Reply:
We don't actually have a `sprk-c-Footer`. Doesn't do anything anyways. Rob and I talked it through and because we had been using the background color utility we went for a modifier. Went with the same way we use the dark background on stepper. Happy to revisit this though.

Reply:
Yeah, lets change this. Modifiers change the defaults that are set. So, having a modifier for something that is never set isn't the way we want to do this as it goes against our CSS standards (BEM).